doc/topics/development/contributing.rst

```diff
@@ -219,201 +219,34 @@ Fork a Repo Guide_>`_ and is well worth reading.
-containing bug fixes or documentation changes should be made against the oldest supported main
-release branch that is affected.
-
-The branch name will be a date-based name such as ``2016.11``.
-
-Bug fixes are made on this branch so that dot release branches can be cut from
-the main release branch without introducing surprises and new features. This
-approach maximizes stability.
-
-Dot Release Branches
-====================
-
-Prior to tagging an official release, a branch will be created when the SaltStack
-release team is ready to tag. The dot release branch is created from a main release
-branch. The dot release branch will be the same name as the tag minus the ``v``.
-For example, the ``2017.7.1`` dot release branch was created from the ``2017.7``
-main release branch. The ``v2017.7.1`` release was tagged at the ``HEAD`` of the
-``2017.7.1`` branch.
-
-This branching strategy will allow for more stability when there is a need for
-a re-tag during the testing phase of the release process and further increases
-stability.
-
-Once the dot release branch is created, the fixes required for a given release,
-as determined by the SaltStack release team, will be added to this branch. All
-commits in this branch will be merged forward into the main release branch as
-well.
-
-Merge Forward Process
-=====================
-
-The Salt repository follows a "Merge Forward" policy. The merge-forward
-behavior means that changes submitted to older main release branches will
-automatically be "merged-forward" into the newer branches.
-
-For example, a pull request is merged into ``2017.7``. Then, the entire
-``2017.7`` branch is merged-forward into the ``2018.3`` branch, and the
-``2018.3`` branch is merged-forward into the ``|repo_primary_branch|`` branch.
-
-This process makes is easy for contributors to make only one pull-request
-against an older branch, but allows the change to propagate to all **main**
-release branches.
-
-The merge-forward work-flow applies to all main release branches and the
-operation runs continuously.
-
-Merge-Forwards for Dot Release Branches
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-
-The merge-forward policy applies to dot release branches as well, but has a
-slightly different behavior. If a change is submitted to a **dot** release
-branch, the dot release branch will be merged into its parent **main**
-release branch.
-
-For example, a pull request is merged into the ``2017.7.2`` release branch.
-Then, the entire ``2017.7.2`` branch is merged-forward into the ``2017.7``
-branch. From there, the merge forward process continues as normal.
-
-The only way in which dot release branches differ from main release branches
-in regard to merge-forwards, is that once a dot release branch is created
-from the main release branch, the dot release branch does not receive merge
-forwards.
-
-.. note::
+When the time comes for a new release which should be every 3 to 4 months
+the release will be cut from the master.
```

Comment:
Let's rephrase this sentence to put the most important information first:

The release will be cut from the master when the time comes for a new release, which should be every 3 to 4 months.